contracts/IGatewayProtocol.sol

```diff
@@ -1,8 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
```

Comment:
Random note. Theres inconsistency in our versioning - the end result is the same but for consistency in our release candidate may as wlel have them all the same.

Reply:
I attempted to lower most of our contracts to `^0.8.0` although that's probably wrong.  Imported contracts are as-is.  I don't think we're using any weird Solidity features so any stable version is fine with me.  I'll include a tool that replaces all versions in `{src/, test/}`.